src/test/java/org/broadinstitute/hellbender/utils/mcmc/GibbsSamplerCopyRatioUnitTest.java

```diff
@@ -0,0 +1,412 @@
+package org.broadinstitute.hellbender.utils.mcmc;
```

Comment:
I think I'll keep this separate as a simple example of the API usage and will have a separate test for the full copy-ratio model.  I considered lumping both the SingleGaussian and CopyRatio Tests together into GibbsSamplerUnitTest but that was quite long and sprawling...I know it's not standard to have two separate Test classes but I think it's warranted here.